LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013
+Copyright (c) 2013-2020 Chan Zuckerberg Initiative
```

Comment:
I think we started working on cellxgene in 2017.

Reply:
Done!